components/renku_data_services/notebooks/api.spec.yaml

```diff
@@ -723,6 +876,158 @@ components:
       - renku.io/projectName
       - renku.io/repository
       type: object
+    SessionPostRequest:
+      properties:
+        project_id:
+          $ref: "#/components/schemas/Ulid"
+        launcher_id:
+          $ref: "#/components/schemas/Ulid"
+        storage:
+          default: 1
+          type: integer
+          description: The size of disk storage for the session, in gigabytes
+        resource_class_id:
+          default: 
+          nullable: true
+          type: integer
+      required:
+        - project_id
+        - launcher_id
+      type: object
+    SessionResponse:
```

Comment:
Which part here represents the sessionId?

Reply:
It's the `name` field.